jwst/extract_1d/ifu.py

```diff
@@ -1064,3 +1076,26 @@ def shift_ref_image(mask, delta_y, delta_x, fill=0):
     temp[..., oslice_y, oslice_x] = mask[..., islice_y, islice_x]
 
     return temp
+
+
+def copy_keyword_info_ifu( source_type, x_center, y_center, slitname , spec: SpecModel):
```

Comment:
I guess the concept of this function is OK, but its version in the module for non-IFU data is specifically copying meta entries from the input `SlitModel` to the output `SpecModel`. Here you're just copying individual, independent variable values that aren't part of a model into `SpecModel` and hence you're forced to send all of those variables into the function as arguments. So I'm not really sure it's worth the effort of having a separate function to do this, as opposed to just having lines 1099-1101 appear up above at line 158.  But I don't have a strong feeling either way. It's OK to keep it too.

Reply:
I removed it. At first I thought we might want to copy a bunch of keywords coming from different locations so I modeled
it after what was in extract.py. But it is just a few simple lines of code now. So it just added unneeded complexity.